jenkins_integration/jenkins_integration.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+//Create curl post request to hit the Jenkins webhook
+$secrets = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/secrets.json'));
+
+$curl = curl_init($secrets->jenkins_url);
+
+//Setup header with authentication
+curl_setopt($curl, CURLOPT_HTTPHEADER, array('Content-Type: application/json',
+    'Authorization: Basic '. base64_encode("$secrets->username:$secrets->api_token"),
+));
+
+//Declare request as a post and setup the fields
+curl_setopt($curl, CURLOPT_POST, true);
+curl_setopt($curl, CURLOPT_POSTFIELDS, array(
+  'token' => $secrets->token,
+));
+
+//Execute the request (could store response if necessary)
+curl_exec($curl);
```

Comment:
I think it'd be nice to have some debugging output here that makes use of the jenkins API response. Do you get a build number in the reply? If so, it could have something like:

> Success! Your Jenkins job has been queued:
> https://jenkins-host.mydomain.com/jobs/<jobname>/builds/<buildnumber>